xml/vt_best_practices.xml

```diff
@@ -479,7 +479,12 @@ always madvise [never]</screen>
      consumption on the &vmhost; by sharing data &vmguest;s have in common. The
      KSM daemon <systemitem class="daemon">ksmd</systemitem> periodically scans
      user memory looking for pages of identical content which can be replaced
-     by a single write-protected page. To enable KSM, run:
+     by a single write-protected page. To enable KSM, start the relative service:
```

Comment:
"relative" should be "ksm service"

Reply:
Thanks for the comments! I see your point, and agree that "relative" is bad. However, re-reading the sentence, I don't think I really like it. I think I'm turning it into: "To enable the KSM service and start it, run the following command:"